Public/Src/Cache/ContentStore/Distributed/Redis/RedisContentLocationStoreConfiguration.cs

```diff
@@ -92,6 +92,11 @@ public class RedisContentLocationStoreConfiguration : LocalLocationStoreConfigur
         /// </remarks>
         public int EvictionPoolSize { get; set; } = 5000;
 
+        /// <summary>
+        /// 
+        /// </summary>
+        public TimeSpan EvictionMinAge { get; set; } = TimeSpan.FromMinutes(30);

```

Comment:
Could you please add a comment, or use <nodoc />?

Reply:
added